markdown/org/docs/patterns/skully/instructions/en.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Skully (the logo): Sewing Instructions"
+---
+
+<Tip>
+
+A lot of the smaller seams (eyes, nose) are best sewn by hand. 
+
+Most of the seams will benefit from being basted before being sewn.
+
+</Tip>
+
+## Step 1: The darts
+
+There are three darts that need to be closed first. Pin and sew them, then remove the bulk and
+give them a good press. Repeat for the other three darts.
+
+## Step 2: The eyes
+
+Sew together the sides of the eye, so you will make a cone. Cut away any bulk. Repeat for the other eye
+
+## Step 3: The nose
+
+Like the eyes, the nose too has parts that need to be sewn together. There are two of these sides that need to be sew together. This creates two connected cones. This is very finicky, so take your time.
```

Comment:
I am bit confused by this bit. Is it the straight edges that are sewn together or the curved edges of the nose?

Reply:
The straight edges. There is a plug-in that shows how, but it is stalled in the plugin-annotations commit.

Reply:
```suggestion
Like the eyes, the nose has seams that need to be sewn together. Sew the two straight seams together creating two connected cones. This is very finicky, so take your time.
```
Would this be better than.